### ISSUE TITLE
how to return Chinese characters  with UTF-8 code

### ISSUE THREAD

**Issue by aixuedegege** [pytorch/serve#943]
1.  When I use torch sever, I return a list in the **postprocess function** of the handler. Each element of the list is a python dictionary and the dictionary value is Chinese characters. Torch sever directly returns a json with the unicode encoding like "\u59d3". Can I control the return using UTF-8? 
2.  In addition, "model-server.jar ”Is there a corresponding document for this? What's his relationship with torch sever?

We look forward to your reply. Thanks a lot.

**Comment by dhanainme** [comment]
> Can I control the return using UTF-8?

Right now we only support Json / unicode. Also, UTF-8 is a particular way of encoding Unicode. Can you please provide some more context on this feature ask ?

> model-server.jar

Frontend of Torchserve is Java / Netty based HTTP server. model-server.jar  is the compiled jar for the same.

**Comment by aixuedegege** [comment]
Hi. Thank you for your reply. I have a ner model and the model return label directly. It is a json. It looks like `{
    "姓名":"程序猿",
    "年龄":"23岁",
    "职业":"码农"
}` .Eache label is the key in json. When I call my model with curl cmd or browser I just see `{"\u59d3\u540d":"\u7a0b\u5e8f\u733f","\u5e74\u9f84":"23\u5c81", "\u804c\u4e1a":"\u7801\u519c"}`. My colleagues received unicode code and need to transfer it to utf-8 to get what the meaning of my model result. I think it's a little inconvenient.

Can you please explain how ts and model-server.jar interact with each other? or by which function ts pass data to Torchserve.
Thanks a lot.

**Comment by lxning** [comment]
Can you check if your model define response content-type as "utf-8"? If not, please try call "set_response_content_type" ([example](https://github.com/pytorch/serve/blob/master/frontend/modelarchive/src/test/resources/models/noop_no_archive/noop/service.py#L85) to see if it can work.

**Comment by msaroufim** [comment]
Hi @aixuedegege did @lxning's workaround work for you? We have a similar issue from way back to work with Japanese characters so if this worked, I'll go ahead and add utf-8 support #821 821

**Comment by aixuedegege** [comment]
> Hi @aixuedegege did @lxning's workaround work for you? We have a similar issue from way back to work with Japanese characters so if this worked, I'll go ahead and add utf-8 support #821

First of all, thank you for @lxning's reply. 
Hi @msaroufim , Sorry, I almost forgot this issue 😄. I do not try the method @lxning  mentioned above because I did not see it. And My colleague transferred the encode style by himself in the background. You can try it and tell me if it works.

**Comment by neoragex2002** [comment]
> 
> 
> Can you check if your model define response content-type as "utf-8"? If not, please try call "set_response_content_type" ( [example](https://github.com/pytorch/serve/blob/master/frontend/modelarchive/src/test/resources/models/noop_no_archive/noop/service.py#L85) ) see if it can work.

No. It doesn't work. No matter what content_type header I have set in context.response (even if charset:utf-8) , the output is always json/unicode... It's really annoying.
